godot_project/autoloads/nano_editor_clipboard/nano_editor_clipboard.gd

```diff
@@ -243,6 +245,13 @@ func _copy_anchor(
 	_copy_structure(in_structure_context, in_shape, out_content)
 
 
+func _copy_particle_emitter(
+	in_structure_context: StructureContext,
+	in_shape: NanoParticleEmitter,
+	out_content: Array[Dictionary]) -> void:
+	_copy_structure(in_structure_context, in_shape, out_content)
+
+
```

Comment:
I've added this to match the existing convention, but why are these methods required? Can't we just call `_copy_structure()` directly in the match above?

Reply:
well, this distinction only exists to make sure we take in consideration exeptional cases that make occur with new classes, in example if we just leave it as is, and we add some new object that requires special setup, like in example anchors needed repaing of springs, we should account for that.

I was not sure at first glance if calling `_copy_structure()` woould create a duplicate of `_parameters`, and inside of it if it create a duplicate of the `template` object
Seems it works and i was concerned for nothing